### PR TITLE
Add gist-vim... pretty crucial imo.

### PR DESCRIPTION
**Pull request by fourq**
# Added options to .vimrc
ref: https://github.com/mattn/gist-vim

## copies the gist url with option '-c'.
```
let g:gist_clip_command = 'pbcopy'
```

## detects filetype for gist
```
let g:gist_detect_filetype = 1
```

## keeps your shit private
```
let g:gist_post_private = 1
```

### CODE REVIEW

**Comment by brentlintner** [comment]
@fourq gist-vim looks awesome! Sorry for the delay, I plan to pull in that commit. :-D

**Comment by fourq** [comment]
ahh, I should have cleaned up this commit. I forgot that I did it off of my forks master. I could make another pr of just that if you'd like, and ignore the rest of my changes.

**Comment by brentlintner** [comment]
@fourq no worries. I can just cherry-pick/rebase. Your call. :-)

**Comment by fourq** [comment]
cherry-pick away my friend. btw SUCH a killer vim setup. ty.

**Comment by brentlintner** [comment]
Ah, why thank you. I've definitely pulled in a lot from lots of others who have paved the way, I must say. Definitely a fan of Pathogen, too. Thanks for the PR! :-)

File: .vimrc [hunk 1]
```diff
@@ -8,6 +8,28 @@ call pathogen#infect()
...
+    \ '<form> lacks "action" attribute',
+    \ '<template> is not recognized!',
+    \ 'discarding unexpected <template>',
+    \ 'discarding unexpected </template>'
+  \ ]
+
+" gist-vim
+let g:gist_clip_command = 'pbcopy' " pbcopy for OSX Only
+let g:gist_detect_filetype = 1
+let g:gist_post_private = 1
```
Comment (fourq) [inline .vimrc]: imo, these are pretty important settings for vim-gist. It makes your gists private, and copies the url to your "clipboard"/buffer.

**Comment by brentlintner** [comment]
Thanks again! vim-gist is sweet. Just been messing around with it, and merged as: https://github.com/brentlintner/vim-settings/commit/500a9b4545591a38e63bed0020ef1d564f625a5f (I just left out the pbcopy since I go back and forth between linux/osx).

**Pull request closed without merging**